src/server/src/main/java/io/cassandrareaper/storage/cassandra/Migration024.java

```diff
@@ -0,0 +1,71 @@
+/*
+ * Copyright 2019-2019 The Last Pickle Ltd
+ *
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package io.cassandrareaper.storage.cassandra;
+
+
+import io.cassandrareaper.core.RepairSegment;
+
+import java.util.UUID;
+
+import com.datastax.driver.core.ConsistencyLevel;
+import com.datastax.driver.core.PreparedStatement;
+import com.datastax.driver.core.ResultSet;
+import com.datastax.driver.core.Row;
+import com.datastax.driver.core.Session;
+
+public final class Migration024 {
+
+  private static final String REPAIR_RUN_TABLE = "repair_run";
+
+  private Migration024() {
+  }
+
+  /**
+   * Populate the segment_active column of the repair_run table based on the value
+   * of the segment_state column.
+   */
+  public static void migrate(Session session, String keyspace) {
```

Comment:
CassandraStorage doesn't call this method. Is it ok?

Reply:
It is called from `CassandraStorage.migrate(…)` through reflection (at `CassandraStorage.java` line 307 in my version of the code).